index.html

```diff
@@ -8,34 +8,41 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body class="body">
+    <header>
+      <nav class="topnav">
+        <a href="index.html"><img class="welcome" src="img/Welcome.png"></a>
+        <a href="#"><img class="Menu" src="img/Icon - Menu.png"></a>
+      </nav>      
+    </header>
     <div class="first-part">
```

Comment:
I see a great effort you have put into this project. An excellent finishing touch at this time would be to make the mobile version responsive until the desktop screen size.

The website at 765px:
![image](https://user-images.githubusercontent.com/20554583/206627698-171199f2-6f71-40c8-b96b-075afe040a43.png)